src/sql/workbench/browser/modelComponents/media/legacycard.css

```diff
@@ -0,0 +1,219 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.model-card {
```

Comment:
Hrm, but this is sharing the class name with the radio card group cards. Are you sure this isn't going to break those too?

https://github.com/microsoft/azuredatastudio/blob/a823e443934589491d95b975c4771f82ceca3ac2/src/sql/workbench/browser/modelComponents/radioCardGroup.component.html#L3

It should be safe to update the legacy cards to use a different class name for the time being so they only pick up the old styles.

Reply:
with this change:
![image](https://user-images.githubusercontent.com/13777222/93541182-8b489380-f90a-11ea-99a5-5155b4971b6a.png)

without this change:
![image](https://user-images.githubusercontent.com/13777222/93541145-723fe280-f90a-11ea-9eff-9b1e9c7266ec.png)

Reply:
but to be safe, let me update the class names

Reply:
just to be absolutely safe, I replaced the class names in the card component and added -legacy suffix. tested and worked as expected